src/Inventory/Asset/InventoryNetworkPort.php

```diff
@@ -404,7 +403,8 @@ private function handleUpdates()
                     }
                 }
 
-                if (count($criteria)) {
+                // force dynamic for NetworkPortAggregate because it can be added manually
+                if (count($criteria) || $dbdata_copy['instantiation_type'] == NetworkPortAggregate::class) {
```

Comment:
I guess that this issue may appears with other instanciation types too. If we just want to set `is_dynamic=true` to manually added ports as long as they match a port listed by the inventory agent, then this condition could be more generic:

```suggestion
                // force dynamic for manually added NetworkPort
                if (count($criteria) || !$dbdata_copy['is_dynamic']) {
```

It requires to add the `is_dynamic` field to the `SELECT` array.

Reply:
done!